packages-common/pluggable-widgets-generator/generators/app/lib/prompttexts.js

```diff
@@ -0,0 +1,151 @@
+const semver = require("semver");
+
+function _promptWidgetProperties(dir, widgetName) {
+    return [
+        {
+            type: "input",
+            name: "name",
+            validate: function(input) {
+                // eslint-disable-next-line wrap-regex
```

Comment:
why?

Reply:
Prettier is acting against this rule. Since we integrated prettier with eslint, we decided to let prettier handle code formatting. Therefore, I removed the eslint rule `"wrap-regex": "error"` from `eslint.js.base.json` from PIW-tools.